### PR TITLE
updates typography to match new styles

### PR DESCRIPTION
**Pull request by stubbornella**
[Finishes #111748002]

### CODE REVIEW

File: src/pivotal-ui/components/mixins.scss [hunk 1]
```diff
@@ -106,7 +106,6 @@
   //pui
   margin-top: $whitespace-xl;
   margin-bottom: $whitespace-l;
-  font-weight: $font-weight-h1;
```
Comment (aredridel) [inline src/pivotal-ui/components/mixins.scss:109]: Why no `font-weight-h1` on an h1 mixin?

**Comment by aredridel** [comment]
Seems straightforward. Bigger structural design I don't quite get -- what the mixin is used for and why parts aren't in there, but I see no reason not to :sheep: if that makes sense

**Comment by stubbornella** [comment]
The mixin is a bit weird, but I promised myself no refactors whilst trying to unblock the team.

**Comment by aredridel** [comment]
Makes sense!

**Comment by aredridel** [comment]
:sheep: then!

**Pull request merged**